source/HailPanel.cpp

```diff
@@ -232,6 +230,29 @@ void HailPanel::Draw()
 					zoom);
 				draw.Add(body);
 			}
+		};
+
+		if(hasFighters)
+			for(const Ship::Bay &bay : ship->Bays())
+				if(bay.side == Ship::Bay::UNDER && bay.ship)
+					draw.Add(Body(*bay.ship, center + zoom * facing.Rotate(bay.point),
+						Point(), facing + bay.facing, zoom));
```

Comment:
We could perhaps use a lambda function for drawing the fighters as we have with the hardpoints.